R/deployments.R

```diff
@@ -11,28 +11,6 @@ saveDeployment <- function(appPath,
                            url,
                            metadata = list()) {
 
-  # TODO(HW): move this logic into deploymentTarget
-  # if there's no new title specified, load the existing deployment record, if
-  # any, to preserve the old title
-  if (is.null(title) || is.na(title) || length(title) == 0 ||
```

Comment:
`saveDeployment()` is only called `deployApp()` where `title` is the `appTitle` returned by `deploymentTitle()`, so I moved the logic there.